integration/lifecycle/lifecycle_test.go

```diff
@@ -206,6 +219,12 @@ var _ = Describe("Lifecycle", func() {
 		By("listing the installed chaincodes and verifying the channel/chaincode definitions that are using the chaincode package")
 		nwo.QueryInstalledReferences(network, "testchannel", chaincode.Label, chaincode.PackageID, network.Peer("Org2", "peer0"), []string{"My_1st-Chaincode", "Version-0.0"})
 
+		By("checking for term files that are written when the chaincode is stopped via SIGTERM")
+		for _, termFile := range termFiles {
+			_, err = os.Stat(termFile)
+			Expect(err).To(BeNil())
+		}
```

Comment:
There's a matcher for this: BeARegularFile

Reply:
Done